config/locales/cy/generic.yml

```diff
@@ -26,6 +26,7 @@ cy:
     list: tsiL
     'no': 'oN'
     menu: uneM
+    none: enoN
```

Comment:
😂 because we have never had a welsh speaker on boarded to translate to welsh, but yes, it is odd and a bit of a wast of time until we do actually translate IMO